docs/content/release_notes/csi_rn_supported_os.md

```diff
@@ -5,9 +5,8 @@ The following table lists operating systems required for deployment of the IBM®
 |Operating system|Architecture|
 |----------------|------------|
 |Red Hat® Enterprise Linux® (RHEL) 7.x|x86, IBM Z®|
-|Red Hat Enterprise Linux CoreOS (RHCOS)|x86, IBM Z®<sup>2</sup>, IBM Power Systems™<sup>1</sup>|
+|Red Hat Enterprise Linux CoreOS (RHCOS)|x86, IBM Z®, IBM Power Systems™<sup>1</sup>|
```

Comment:
```suggestion
|Red Hat Enterprise Linux CoreOS (RHCOS)|x86, IBM Z, IBM Power Systems™<sup>1</sup>|
```
do we need to include the same trademark more than once per page?

Reply:
no. removing. 
unfortunately I can't run MD through my trademark scanner.
DONE